docs/manifest_validation.md

```diff
@@ -0,0 +1,14 @@
+## Manifest validation
+
+Manifest validation is a feature that validate YAML structure definition for task.
+It checks structure, types and related entities.
+
+### How to use
+
+Simply run `spryker-sdk sdk:validate:task` to validate task yaml definition.
+
+### How to fix
+
+ - Task [conventions](conventions.md#Task), [structure](task.md#Task)
```

Comment:
Task validation displays all the errors in the output. Use our documentation below to learn more about task and task set structure and conventions:
- Link 1
- Link 2
- Link N